dags/myla311/download-data.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash 
+
+set -e 
+
+[ -e /tmp/service-requests.csv ] && rm /tmp/service-requests.csv
+
+curl https://data.lacity.org/api/views/69nn-wkid/rows.csv?accessType=DOWNLOAD \
+     -u '$SOCRATA_USERNAME:$SOCRATA_PASSWORD' \
```

Comment:
Looks like you can pass environment variables into bash operators via the command args. We can probably use that to securely store the login info in the airflow db.